ethstats/ethstats.go

```diff
@@ -546,7 +571,10 @@ func (s *Service) readLoop(conn *websocket.Conn) {
 			log.Info("Ping from websocket server", "msg", msg)
 			// Primus server might want to have a pong or it closes the connection
 			var serverTime = fmt.Sprintf("primus::pong::%d", time.Now().UnixNano()/int64(time.Millisecond))
-			conn.WriteJSON(serverTime)
+			err := conn.WriteMessage(websocket.TextMessage, []byte(serverTime))
```

Comment:
why this change?

Reply:
This was a change of the #1033. Thought that maybe was important, and I was not seeing the why